contentcuration/kolibri_content/models.py

```diff
@@ -108,6 +108,9 @@ class ContentNode(MPTTModel):
     # Added legacy fields
     license = models.ForeignKey('License', null=True, blank=True)
 
+    # A JSON Dictionary of properties to configure loading, rendering, etc. the file
+    options = JSONField(default={})
```

Comment:
Do we need to rerun an export on all content databases to be compatible with Kolibri?

Reply:
No - Kolibri's content database import is designed to be completely backwards compatible with older database versions, so if this field is missing, and Kolibri 0.14 imports it, it will just set it to the empty default.